File: user_componet/src/main/debug/java/ZhddApplication.java
import android.app.Application;

import com.billy.cc.core.component.CC;

/**
 * @author xiangyao
 */
public class ZhddApplication  extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        CC.enableVerboseLog(true);
        CC.enableDebug(true);
        CC.enableRemoteCC(true);
    }
}

File: app/src/main/java/com/juju/zhdd/ZhddApplication.kt
package com.juju.zhdd

import android.app.Application
import com.billy.cc.core.component.CC

class ZhddApplication:Application() {


    override fun onCreate() {
        super.onCreate()
        CC.enableVerboseLog(true)
        CC.enableDebug(true)
        CC.enableRemoteCC(true)
    }
}